scripts/run_e2e_tests.sh

```diff
@@ -115,20 +117,32 @@ trap cleanup EXIT
 # Argument passed to feconf.py to help choose production templates folder.
 DEV_MODE=true
 RUN_ON_BROWSERSTACK=False
+
+# Currently, the community dashboard page is disabled.
+community_dashboard_status_variable="COMMUNITY_DASHBOARD_ENABLED = False"
```

Comment:
We need an issue to remove these once the community dashboard page is enabled.

Reply:
I've created an issue https://github.com/oppia/oppia/issues/8166.